### DISCUSSION TITLE
Mikro-orm pagination

### DISCUSSION THREAD

**Discussion by rita346** [mikro-orm/mikro-orm#6010] [Q&A]
Hi,

I am currently working with MikroORM for my NestJS project and am looking for a library or solution that can handle pagination in a similar way to how nestjs-typeorm-paginate works with TypeORM. Specifically, I am interested in a solution that can automatically provide detailed pagination information, including links, metadata, and pagination details, without requiring manual calculation or additional coding on my part.

Is there an existing library or recommended approach for MikroORM that supports this functionality? If so, could you provide some guidance on how to integrate it or direct me to the relevant resources?

Thank you!

Here's my code :
import {Cascade, Collection, Entity, ManyToOne, OneToMany, Opt, PrimaryKey, Property} from "@mikro-orm/core";
import {Book} from "../../book/entities/book.entity";

@Entity({tableName:'author'})
export class Author {
    @PrimaryKey({autoincrement: true})
    id?: number;

    @Property()
    name?: string;

    @Property()
    email?: string;

    @Property({ nullable: true })
    age?: number;

    @Property()
    termsAccepted?: boolean & Opt = false;

    @Property({ nullable: true, })
    born?: Date;

    @OneToMany(() => Book, b => b.author, { cascade: [Cascade.REMOVE] })
    books = new Collection<Book>(this);

    @ManyToOne(() => Book, { nullable: true })
    favouriteBook?: Book;

    @Property()
    createdAt?: Date & Opt = new Date();

    @Property({ onUpdate: () => new Date() })
    updatedAt?: Date & Opt = new Date();

    constructor(name: string, email: string) {
        this.name = name;
        this.email = email;
    }

}
Let's say I want to get all the authors in my database but with pagination how to do it?

**Comment by Maxttier** [comment]
https://www.npmjs.com/package/@emulienfou/nestjs-mikro-orm-paginate
